guide/app/layout.tsx

```diff
@@ -106,16 +172,30 @@ export function Guide({
 	return (
 		<div className="xl:container mx-auto xl:grid xl:grid-cols-5 gap-10 px-8 relative">
 			<header className="bg-zinc-900 xl:bg-transparent sticky top-0 max-h-screen z-10 flex flex-col">
-				<div className="py-2 xl:py-4">
+				<div className="py-2 xl:py-4 flex items-center">
 					<Link
-						className="font-mono inline-block py-4 text-[.25rem] leading-[.25rem] xl:text-[.35rem] xl:leading-[.40rem] whitespace-pre"
+						className="flex-1 font-mono inline-block py-4 text-[.25rem] leading-[.25rem] xl:text-[.35rem] xl:leading-[.40rem] whitespace-pre"
 						title="Conform"
 						to="/"
 					>
 						{logo}
 					</Link>
+					<select
+						className="bg-zinc-900 text-xs rounded-md py-1"
+						defaultValue={language.code}
+						onChange={(e) => {
+							const selectedLanguage = getLanguage(e.target.value);
```

Comment:
does the same thing, but... semantics
```suggestion
							const selectedLanguage = getLanguage(e.currentTarget.value);
```

Reply:
Fixed to currentTarget. Thank you.